README.md

```diff
@@ -19,14 +19,16 @@ The standard gradle pipeline is the simplest way to get up an running quickly. I
-| STANDARD_GRADLE_FLAGS   	| -s --no-daemon -PnoMavenLocal --refresh-dependencies --console=plain 	| Default Gradle flags that will be appended to all Gradle commands                                                                                         |
-| EXTRA_GRADLE_TEST_FLAGS 	|                                                                      	| Flags that will appended to test tasks.                                                                                                                	|
-| RELEASE                 	|                                                                      	| The name that will be appended to release build artifacts. By default an release candidate will be created from this unless the value "final" is used. 	|
-| DEV_REGEX                 | develop                                                               | Branch(es) jobs will be run from when new commits are made. For example, if it's desired to run jobs from from `v2-develop` and `v3-develop` branches, this variable can be set to `'^v3-develop\|$^v2-develop$'`
-
+| Variable                  | Pre-Loaded**| Default Value                                                        	                | Description                                                                                                                                            	|
+|-----------------------	|-----------|-----------------------------------------------------------------------------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| DEFAULT IMAGE           	|           | openjdk:11                                                                            | The base docker image used to run all included jobs. Jobs can also be further customized by specifying a different image for a specific job.           	|
+| STANDARD_GRADLE_FLAGS   	|           | -s --no-daemon -PnoMavenLocal --refresh-dependencies --console=plain  (-PsafeTest)	| Default Gradle flags that will be appended to all Gradle commands (Will include -PsafeTest when set)                                                                                        |
+| DEV_REGEX                 |           | develop                                                                               | Branch(es) jobs will be run from when new commits are made. For example, if it's desired to run jobs from from `v2-develop` and `v3-develop` branches, this variable can be set to `'^v3-develop\|$^v2-develop$'`
+| SAFE_TEST                 |X          | false                                                                                 | Boolean on whether to run the build pipeline as a test before actually deploying, when set to \"true\" the build will not publish or deploy and artifacts.|
```

Comment:
Would it be clearer to use true / false instead of "X" to indicate whether a variable is preloaded?